### PR TITLE
Patched workflow for NodeJS

### PR DESCRIPTION
**Pull request by carlopi**

### CODE REVIEW

File: .github/workflows/NodeJS-dispatched.yml [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.inputs.ref }} 
+
+      - name: Patch
+        shell: bash
+        run: |
+          touch .github/patches/duckdb/nodejs.patch
+          git apply .github/patches/duckdb/nodejs.patch
```
Comment (samansmink) [inline .github/workflows/NodeJS-dispatched.yml:221]: should this have a `--allow-empty` to allow running this without a patch?

**Review by samansmink — COMMENTED** [review]
added a minor comment but the rest looks good, ideally we would want to have this in a single workflow as there is a lot of duplication ofc, but i understand that that could be something to look into after getting the node 0.9.0 binaries out.

File: .github/workflows/NodeJS-dispatched.yml (hunk 1 above)
Comment (carlopi) [reply .github/workflows/NodeJS-dispatched.yml:221]: I am not yet sure this makes sense at all, it's mostly a one-off bandaid, likely to be deleted afterwards while finding a proper long term solution to this.

Comment (samansmink) [reply .github/workflows/NodeJS-dispatched.yml:221]: yea fair enough, 🩹 that 💩 up!

**Comment by carlopi** [comment]
Closing this since work is underway for a more proper fix

**Pull request closed without merging**